context/context.go

```diff
@@ -26,6 +26,7 @@ import (
 
 func init() {
 	smfContext.NfInstanceID = uuid.New().String()
+	os.Setenv("SMF_ID", smfContext.NfInstanceID)
```

Comment:
Would be interested in knowing if this is good practice to set & use environment variables.